lib/server/readMetadata.js

```diff
@@ -345,6 +322,13 @@ function generateMetadataBlog() {
   const metadatas = [];
 
   let files = glob.sync(CWD + '/blog/**/*.*');
+  if (!files || files.length == 0) {
```

Comment:
Same here. This was also removed in https://github.com/facebook/Docusaurus/pull/291

Reply:
removed